perma_web/perma/tasks.py

```diff
@@ -973,33 +948,29 @@ def _proxy_request(self):
             print("Finished fetching URL.")
 
             # Get a fresh copy of the page's metadata, if possible
-            with browser_running(link, browser):
-                inc_progress(capture_job, 1, "Retrieving page metadata (after onload)")
-                add_thread(thread_list, get_metadata, args=(
-                    page_metadata,
-                    link,
-                    browser
-                ))
+            dom_tree = get_dom_tree(browser)
+            get_metadata(page_metadata, dom_tree)
```

Comment:
I took a pass at a Forbes handler that attempts to click and then waits for the redirect rather than a fixed timeout. Since it's a bit longer, I moved it to a separate file.

It's basically free to add a check after the scroll, so we can do both for now -- I threw in an extra one. In the long run we should just stop the browser from crashing ...